app/controllers/api/v1/admin/reviews_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Admin
+      class ReviewsController < Api::V1::Admin::AdminApiController
+        def index
+          @reviews = Review.all
+        end
+
+        def destroy
+          @review = Review.find(params[:id])
+          @review.destroy
+        end
+
+        def create
+          ActiveRecord::Base.transaction do
+            @review = Review.create!(review_params.merge(reviewer_id: current_user.id))
+            review_action_items = params[:review_action_items].map do |item|
+              ReviewActionItem.create!(
+                create_review_action_item(item).merge(review_id: @review.id)
+              )
+            end
+            @review.review_action_item = review_action_items
```

Comment:
We should use `nested_attributes` https://api.rubyonrails.org/classes/ActiveRecord/NestedAttributes/ClassMethods.html